front/app/containers/Admin/users/UserTableRow.tsx

```diff
@@ -135,11 +234,59 @@ const UserTableRow = ({
         )}
       </RegisteredAt>
       <Td>
-        <Toggle checked={isUserAdmin} onChange={toggleAdmin} />
+        <FormattedMessage {...statusMessage} />
       </Td>
       <Td>
         <MoreActionsMenu showLabel={false} actions={actions} />
       </Td>
+
+      <Modal
```

Comment:
If it's easy, it may make sense to move it to a separate component and add a little test for it. If not easy, it's fine as it is.

Reply:
This is one probably best handled by an e2e test. We can add those separately to verify that the entire flow works

Reply:
Yes, make sense. But maybe moving the new code to a separate component would still make sense if it's easy. Because this component has become quite big.

Reply:
Alright, on it